iree/hal/cuda/graph_command_buffer.c

```diff
@@ -30,7 +30,7 @@ typedef struct {
   void* current_descriptor[];
 } iree_hal_cuda_graph_command_buffer_t;
 
-static const size_t max_binding_count = 64;
+#define max_binding_count 64
```

Comment:
nit: this could still be the typed value, or if a macro then style as `IREE_HAL_CUDA_MAX_BINDING_COUNT`